run_scans.py

```diff
@@ -107,10 +107,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, scanmin, scanmax, stepSize
     cmd.append( "--scanmax=%d"%(scanmax) )
     cmd.append( "--nevts=%d"%(nevts) )
     cmd.append( "--throttle=%i"%(throttle) )
-    if stepSize > 0:
-      step = stepSize
-      if (step + scanmin > scanmax):
-        step = scanmax - scanmin
+    if stepSize:
       cmd.append( "--stepSize=%d"%(step) )
```

Comment:
`stepSize` is not boolean but integer, example, how is `stepSize = 20` treated?  Please change to something like `stepSize > 0`

Also `step` looks like it has the value of 0 here, why use another variable at all? You have `stepSize`, use that.